lib/utilities/paginator.rb

```diff
@@ -0,0 +1,81 @@
+module Utilities
```

Comment:
Woot! This is called a service class. Is a Ruby class that is not tied to the database used to do some heavy computational lifting.

Service classes are great and really help clean up your code elsewhere like in your view above
:+1: :100:  for this pattern